stdlib/public/core/KeyPath.swift

```diff
@@ -173,7 +173,8 @@ public class AnyKeyPath: Hashable, _AppendKeyPath {
   }
 
   internal func isClass(_ item: Any.Type) -> Bool {
-    // Displays "warning: 'is' test is always true" at compile time, but that's not actually the case.
+    // Displays "warning: 'is' test is always true" at compile time,
+    // but that's not actually the case.
```

Comment:
I’d thought that this has to do with Obj-C bridging shenanigans, whereby anything can be wrapped?

Reply:
Yeah, the warning is true, modulo runtime bugs. But I think we can remove all of these accessors (isPureStructKeyPath, isClass, isTuple) when the inline offset is computed during pattern instantiation.